README.md

```diff
@@ -244,6 +244,53 @@ This will analyze last 10k blocks and give you 25/50/75/99th/Max percentiles for
 
 `-tp 0.99` requests the 99th tip percentile across all the transaction in one block and calculates 25/50/75/99th/Max across all blocks
 
+### Block stats
+If you need to get some insights into network stats and create a realistic load/chaos profile with simulators (`anvil` as an example), you can use `stats` CLI command
+
+Edit your `seth.toml`
+```
+[[networks]]
+name = "MyCustomNetwork"
+chain_id = "11155111"
+urls_secret = ["..."]
+
+[block_stats]
+rpc_requests_per_second_limit = 5
+```
+
+Then check the stats for the last N blocks
+```bash
+seth -n MyCustomNetwork blocks -s -10
+```
+
+To check stats for the interval (A, B)
+```bash
+seth -n MyCustomNetwork blocks -s A -e B
+```
+
+Results can help you to understand if network is stable, what is avg block time, gas price, block utilization and transactions per second
+```toml
+Stats:
+perc_95_tps = 8.0
+perc_95_block_duration = '3s'
+perc_95_block_gas_used = 1305450
+perc_95_block_gas_limit = 15000000
+perc_95_block_base_fee = 25000000000
+avg_tps = 2.433333333333333
+avg_block_duration = '2s' 
+avg_block_gas_used = 493233
+avg_block_gas_limit = 15000000
+avg_block_base_fee = 25000000000
```

Comment:
Can we also include block size? Running into a need of similar stats in ccip. Block Size will help here.

Reply:
Okay, can include block size. Also, I think it's useful to dump in TOML how many blocks we analyzed. Will add both